make_reports.py

```diff
@@ -62,11 +64,15 @@ def get_options():
 LABEL = 'Outstanding Targets'
 PLANNING_LIMIT = opt.planning_limit
 TASK_DATA = os.path.join(os.environ['SKA'], 'data', 'aca_lts_eval')
+MANVR_ERROR = 60
+
+if opt.creep:
```

Comment:
In reality the maneuver error for man_angle < 5 deg can be up to 20 arcsec, with 90% percentile around 10 arcsec.  Not sure how this precisely fits into minisausage usage of this, but bear that in mind.

Also, use the ternary expression here, this is a poster-child case for that.

Reply:
Thanks!  I think the real win in this code is just that the probabilities get evaluated with the 60 arcsec boxes.  But there's no reason not to be more physical about this; minisausage is using the expected maneuver angle both to estimate the useful area of the CCD for acq boxes and to determine the somewhat-expected distance to the closest hot pixel in the acq box.  Setting to 10 for that purpose seems fine.